openwebif/api.py

```diff
@@ -156,6 +168,22 @@ def default_all(self):
         self.status_info = {}
         self.is_recording_playback = False
 
+    def send_message(self, text: str, message_type: MessageType = MessageType.INFO):
+        """
+        Sends a message to the TV screen.
+
+        :param text: The message to display
+        :param message_type: The type of message (0 = YES/NO, 1 = INFO, 2 = WARNING, 3 = ERROR)
+        :return: True if successful, false if there was a problem
+        """
+
+        url = self._base + \
+            URL_MESSAGE.format(
+                str(self.message_display_timeout or -1), message_type.value, text)
```

Comment:
Does `text` need to be URL encoded?

Reply:
Oh, yeah, of course, sorry for that. Is fixed.